### PR TITLE
add accuracy summary

### PR DESCRIPTION
**Pull request by RUIJIEZHONG66166**

### CODE REVIEW

**Comment by RUIJIEZHONG66166** [comment]
Ondemand test pass.
Log: https://github.com/intel/torch-xpu-ops/actions/runs/9659586596

File: .github/scripts/inductor_accuracy_summary.py [hunk 1]
```diff
@@ -0,0 +1,368 @@
...
+
+# def validate_csv_files(csv_file):
+#     df = pd.read_csv(csv_file,header=None)
+#     expected_header = ['dev', 'name', 'batch_size', 'accuracy', 'calls_captured','unique_graphs','graph_breaks','unique_graph_breaks']
+#     if df.iloc[0].tolist() == expected_header:
+#         print(f"{csv_file} Expected csv file")
+#         return True
+#     else:
+#         print(f"{csv_file} file no right header!!!")
+#         return False
```
Comment (chuanqi129) [inline .github/scripts/inductor_accuracy_summary.py:45]: clean up unnecessary lines

**Review by chuanqi129 — APPROVED** [review]
Overall LGTM, 2 questions
1. rename the performance report excel with `_E2E_Test_Perf_Report.xlsx`
2. Does this log summary script just process 1 suite for once call? How about multiple suites scenario?

File: .github/scripts/inductor_accuracy_summary.py (hunk 1 above)
Comment (RUIJIEZHONG66166) [reply .github/scripts/inductor_accuracy_summary.py]: Done

**Comment by RUIJIEZHONG66166** [comment]
> Overall LGTM, 2 questions
> 
> 1. rename the performance report excel with `_E2E_Test_Perf_Report.xlsx`
> 2. Does this log summary script just process 1 suite for once call? How about multiple suites scenario?

**Comment by RUIJIEZHONG66166** [comment]
> Overall LGTM, 2 questions
> 
> 1. rename the performance report excel with `_E2E_Test_Perf_Report.xlsx`
[Ruijie] will do it later.
> 2. Does this log summary script just process 1 suite for once call? How about multiple suites scenario?
[Ruijie] it should cover multiple suites for once call.

**Pull request merged**